### DISCUSSION TITLE
Camera group in one URL

### DISCUSSION THREAD

**Discussion by tmeekes** [mrlt8/docker-wyze-bridge#317] [Q&A] [answered]
Does the wyze bridge support creating a single URL for a device group? So that you can see multiple cameras on one URL?

**Comment by mrlt8** [comment, marked as answer]
No. The bridge just passes the video stream from the camera to [rtsp-simpe-server](https://github.com/aler9/rtsp-simple-server) which then creates the HLS/RTSP/RTMP streams for each camera. 

You may be able to create your own html page to display multiple HLS streams on one page though.
